src/Controls/src/Core/BindablePropertyConverter.cs

```diff
@@ -123,6 +107,18 @@ BindableProperty ConvertFrom(Type type, string propertyName, IXmlLineInfo linein
 			return bp;
 		}
 
+		[UnconditionalSuppressMessage("TrimAnalysis", "IL2057:TypeGetType",
+			Justification = "The converter is only used when parsing XAML at runtime. The developer will receive a warning " +
+				"saying that parsing XAML at runtime may not work as expected when trimming.")]
```

Comment:
Do you have an example of what the warning looks like?

Reply:
The warning is `TrimmerConstants.XamlRuntimeParsingNotSupportedWarning`: _"Loading XAML at runtime might require types and members that cannot be statically analyzed. Make sure all of the required types and members are preserved."_